### PR TITLE
[Tabs][Addimages][1580403]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Added images. Please review.

### CODE REVIEW

File: msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md [hunk 1]
```diff
@@ -73,7 +81,17 @@ The application must reference the [Microsoft Teams JavaScript client SDK](/java
...
+
+The following image displays the tab content with gray icon:
+
+<img src="~/assets/images/tab-images/configure-tab-with-gray.png" alt="Configure tab with select gray" width="400"/>
+
+The following image displays the tab content with red icon:
+
+<img src="~/assets/images/tab-images/configure-tab-with-red.png" alt="Configure tab with select red" width="400"/>
+
+Choosing the relative button fires either `saveGray()` or `saveRed()`, and invokes the following:
```
Comment (KirtiPereira) [inline msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md:94]:
```suggestion
Choosing the relative button triggers either `saveGray()` or `saveRed()`, and invokes the following:
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Rajeshwari-v - please validate and commit.

**Review by Wajeed-msft — APPROVED** [review]

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Pull request merged**